src/main/webapp/resources/js/pages/analysis/AnalysisCompletePage.jsx

```diff
@@ -0,0 +1,144 @@
+  import("./components/AnalysisPhylogeneticTree")
+);
+const AnalysisProvenance = React.lazy(() =>
+  import("./components/AnalysisProvenance")
+);
+const AnalysisSistr = React.lazy(() => import("./components/AnalysisSistr"));
+const AnalysisDelete = React.lazy(() =>
+  import("./components/settings/AnalysisDelete")
+);
+const AnalysisDetails = React.lazy(() =>
+  import("./components/settings/AnalysisDetails")
+);
+const AnalysisSamples = React.lazy(() =>
+  import("./components/settings/AnalysisSamples")
+);
+const AnalysisSettingsContainer = React.lazy(() =>
+  import("./components/settings/AnalysisSettingsContainer")
+);
+const AnalysisShare = React.lazy(() =>
+  import("./components/settings/AnalysisShare")
+);
+
+/**
+ * React component to facilitate the nested routing needed for the complete
+ * Analysis.
+ * @returns {JSX.Element}
+ */
+function AnalysisOutlet() {
+  return (
+    <AnalysisOutputsProvider>
+      <Suspense fallback={<ContentLoading />}>
+        <Outlet />
+      </Suspense>
+    </AnalysisOutputsProvider>
+  );
+}
+
+/*
+ * The following code sets the key which should
+ * be highlighted by default on page load if the
+ * analysis has completed. If an analysis is a type
+ * of sistr, bio_hansel, phylogenomics, or mentalist,
+ * then a special view is add as the default,
+ * otherwise the output files tab key is set.
+ * If the analysis is not completed and not errored
+ * the the settings tab is the default key. If the
+ * job has errored then the error tab key is set as
+ * the default.
+ */
+const getPipelineType = ({ treeDefault, analysisViewer }) =>
+  analysisViewer === "sistr"
+    ? ANALYSIS.SISTR
+    : analysisViewer === "biohansel"
+    ? ANALYSIS.BIOHANSEL
+    : analysisViewer === "tree" && treeDefault
+    ? ANALYSIS.TREE
+    : ANALYSIS.OUTPUT;
+
+export default function AnalysisCompletePage() {
+  const { analysisContext } = useContext(AnalysisContext);
+  const type = getPipelineType(analysisContext);
+
+  const DEFAULT_URL = setBaseUrl(`/analysis/:id`);
+
+  let component;
+  if (type === "sistr") {
+    component = <AnalysisSistr />;
+  } else if (type === "biohansel") {
+    component = <AnalysisBioHansel />;
+  } else if (type === "tree") {
+    component = <AnalysisPhylogeneticTree />;
+  }
+
+  return (
+    <Space
+      direction="vertical"
+      size="large"
+      style={{ width: `100%`, padding: SPACE_LG }}
+    >
+      <AnalysisMenu type={type} />
+      <Routes>
+        <Route path={DEFAULT_URL} element={<AnalysisOutlet />}>
+          {type === "output" ? (
+            <>
+              <Route index element={<AnalysisOutputFiles />} />
+              <Route path={ANALYSIS.OUTPUT} element={<AnalysisOutputFiles />} />
+              <Route
+                path={ANALYSIS.PROVENANCE}
+                element={<AnalysisProvenance />}
+              />
+            </>
+          ) : (
+            <>
+              <Route path={`sistr/*`} element={component} />
```

Comment:
Need to remove hardcoded `sistr/*`, I suggest to set a `component_path` variable in if statement starting on line 84 and use that instead.

Reply:
For sistr this should probably be `${ANALYSIS.SISTR}/*`

Reply:
Updated in 270bfc265e77193d5a5c3c907a65c8bda246bd73